src/ontology/phenio.Makefile

```diff
@@ -3,4 +3,28 @@
 ## If you need to customize your Makefile, make
 ## changes here rather than in the main Makefile
 
+# Replace the assembly of phenio-full.owl with a
+# similar process including subq reconstruction
 
+# Constants
+SUBQ_QUERY_PATH=                $(SPARQLDIR)/subq_construct.sparql
+UPDATE_QUERY_PATH=              ./subq_update.sparql
```

Comment:
All sparql (UPDATE, SELECT, ASK) should be in the $(SPARQLDIR)

Reply:
unless, if generated on the fly as it seems to be the case here, then stick in $(TMPDIR)